recipes/galgebra/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "galgebra" %}
+{% set version = "0.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pygae/{{ name }}/archive/v{{ version }}.zip
+  sha256: 87a9341dac51d4a81e1d4ccad9f0a96cc6a9dc9cbc4a1235c2872c210275bc54
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip   
+  run:
+    - python
+    - sympy
+
+test:
+  imports:
+    - galgebra
+
+about:
+  home: http://galgebra.readthedocs.io
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Symbolic Geometric Algebra/Calculus package for SymPy.'
+
+  description: |
+    Symbolic Geometric Algebra/Calculus package for SymPy. BSD License.
+  doc_url: http://galgebra.readthedocs.io
+  dev_url: https://github.com/pygae/galgebra
+
+extra:
+  recipe-maintainers:
+    - utensil
+    - hugohadfield
```

Comment:
@hugohadfield are you OK with being maintainer

Reply:
Yes, I am